api/schemas/serializers.py

```diff
@@ -23,6 +36,34 @@ class Meta:
         type_ = 'schemas'
 
 
+class RegistrationSchemaFormBlockSerializer(JSONAPISerializer):
+
+    id = IDField(source='_id', read_only=True)
+    type = TypeField()
+    page = ser.CharField(max_length=255, read_only=True)
+    section = ser.CharField(max_length=255, read_only=True)
+    help_text = ser.CharField(read_only=True, allow_blank=True)
+    block_id = ser.CharField(max_length=255)
+    block_type = ser.ChoiceField(choices=osf.models.metaschema.FORMBLOCK_TYPES)
```

Comment:
This (and `size`) can probably be `CharField`s with `read_only` -- it's unlikely this model will ever be editable through the API.